src/app/static/src/js/components/reportLog/runningReportDetails.vue

```diff
@@ -133,28 +137,44 @@
         },
         methods: {
             getLogs: function () {
-                api.get(`/running/${this.reportKey}/logs/`)
-                    .then(({data}) => {
-                        this.reportLog = data.data
-                        this.error = "";
-                        this.defaultMessage = "";
-                    })
-                    .catch((error) => {
-                        this.error = error;
-                        this.defaultMessage = "An error occurred when fetching logs";
-                    });
+                if (this.reportKey) {
+                    api.get(`/running/${this.reportKey}/logs/`)
+                        .then(({data}) => {
+                            this.reportLog = data.data;
```

Comment:
A nice (optional?) enhancement would be equivalent of `tail -f` (cf. Buildkite's "Follow" for build logs). Not sure how it would interact (if at all) with Vue, but if you just call `textarea.scrollTop = textarea.scrollHeight;` whenever updating `reportLog` then it should automatically scroll to bottom (or at least it did via browser developer console). Would also serve to show user that something is actually happening. But maybe for another ticket.